src/modules/keyboardmanager/KeyboardManagerEditor/Resources.resx

```diff
@@ -169,7 +169,8 @@
     <value>Target App:</value>
   </data>
   <data name="EditKeyboard_OrphanedDialogTitle" xml:space="preserve">
-    <value>The following keys have been reassigned and you won't be able to use them for their original assignment:</value>
+    <value>Warning: The following keys are currently not assigned:</value>
```

Comment:
I get the simplicity of this wording, but feels like it's way less informant than previous. I don't have better, though..
Maybe:

```suggestion
    <value>Warning: The following keys are currently reassigned:</value>
```

If 'reassigned' was not the problem with translation

Reply:
Simple / short wording is better. 

“The follow keys don’t have assignments”?